MainModule/Client/UI/Default/UserPanel.lua

```diff
@@ -200,7 +200,7 @@ return function(data)
 				showItems()
 			else
 				items:Add("TextLabel", {
-					Text = "Cannot edit this table in-game";
+					Text = "Cannot edit this table in the experience";
```

Comment:
Change
`Text = "Cannot edit this table in the experience";`
To
`Text = "Cannot edit this table while in an experience";`

Reply:
sure